src/main/java/spoon/reflect/declaration/CtType.java

```diff
@@ -34,6 +34,13 @@
 	String INNERTTYPE_SEPARATOR = "$";
 
 	/**
+	 * Returns the simple (unqualified) name of this element. If the type is
+	 * a local type, the name starts with its identifier.
+	 */
```

Comment:
Returns the simple (unqualified) name of this element. Following the compilation convention, if the type is a local type, the name starts with a numeric prefix (e.g. local class Foo has simple name 1Foo).